auslib/db.py

```diff
@@ -1743,6 +1743,27 @@ def hasPermission(self, username, thing, action, product=None, transaction=None)
         return True
 
 
+class Dockerflow(AUSTable):
+    def __init__(self, db, metadata, dialect):
+        self.table = Table('dockerflow', metadata, Column('watchdog', Integer))
+        AUSTable.__init__(self, db, dialect, history=False)
+
+    def getDockerflowEntry(self, transaction=None):
+        return self.select(transaction=transaction)[0]
+
+    def incrementWatchdogValue(self, changed_by, transaction=None, dryrun=False):
+        try:
+            what = self.getDockerflowEntry()
+            old_data_version = what['data_version']
```

Comment:
Hm, I just realized that versioning is enabled on this table. You can just flip this off (pass versioned=False to the AUSTable constructor) and you don't need to worry about data_version handling.